openfisca_web_api_preview/openAPI.yml

```diff
@@ -0,0 +1,253 @@
+swagger: "2.0"
+info:
+  title: "{CONTRY_PACKAGE_NAME} Web API"
+  description: "The OpenFisca Web API lets you get up-to-date information and formulas pertaining to the {CONTRY_PACKAGE_NAME} legislation e.g. minimum wage, state scholasrship, social benefits, income taxes…"
+  version: "0.1.0"
+  termsOfService: "https://doc.openfisca.fr/publishing.html"
+  contact:
+    email: "contact@openfisca.fr"
+  license:
+    name: "AGPL"
+    url: "https://www.gnu.org/licenses/agpl-3.0.html"
+host: null
+schemes:
+  - "https"
+tags:
+- name: "Parameters"
+  description: "A parameter is a numeric property of the legislation which can evolve over time."
+  externalDocs:
+    description: "Find out more"
+    url: "https://doc.openfisca.fr/parameters.html"
+- name: "Variables"
+  description: "A variable depends on a person, or an entity (e.g. zip code, salary, income tax)."
+  externalDocs:
+    description: "Find out more"
+    url: "https://doc.openfisca.fr/variables.html"
+paths:
+  /parameters:
+    get:
+      tags:
+      - "Parameters"
+      summary: "List all available parameters"
+      operationId: "getParameters"
+      produces:
+      - "application/json"
+      responses:
+        200:
+          description: "The list of parameters is sent back in the response body"
+          headers :
+            Country-Package :
+              description : "The name of the country package served by the API"
+              type: "string"
+            Country-Package-Version:
+              description: "The version of the country package served by the API"
+              type: "string"
+          schema:
+            $ref: '#/definitions/Parameters'
+  /parameter/{parameterID}:
+    get:
+      tags:
+      - "Parameters"
+      summary: "Gets information about a specific parameter"
```

Comment:
Conjugation inconsistency: the previous `summary` uses infinitive. I'm in favour of infinitive, as it is not the endpoint that _gets information_, but the user.